### PR TITLE
AB#179 Add link to Etherscan

### PR DESCRIPTION
**Pull request by ktothova**
- added link to etherscan 
- added useHead
- changed favicon.ico to our logo

### CODE REVIEW

File: pages/project/login.vue [hunk 1]
```diff
@@ -13,6 +13,10 @@ definePageMeta({
   },
 });
 
+useHead({
+  title: 'Project page - Login',
```
Comment (bran0h) [inline pages/project/login.vue]: Graph explorer - Login

File: components/project/sections/NftInfo.vue [hunk 2]
```diff
@@ -43,13 +65,17 @@
...
 import type { DataTableColumn } from 'naive-ui';
-import { NSkeleton, NDataTable } from 'naive-ui';
+import { NSkeleton, NDataTable, NButton } from 'naive-ui';
 import { $obtain } from '@kodadot1/minipfs';
 import ServerErrorComponent from '../other/ServerError.vue';
 import TruncatedAddressComponent from '../other/TruncatedAddressComponent.vue';
 import type { NFTDTO, TransactionDTO, TransactionSearchRequestDTO, TransactionSearchResponseDTO } from '~/types/dtos';
 const transactionsTable = ref<InstanceType<typeof NDataTable> | undefined>();
 
+useHead({
```
Comment (bran0h) [inline components/project/sections/NftInfo.vue:58]: Graph explorer - NFT

File: components/project/sections/NftInfo.vue [hunk 3]
```diff
@@ -7,7 +7,29 @@
         <img v-else class="w-full" :src="nftData.image" />
       </div>
       <div class="md:col-span-2">
-        <n-p class="hidden text-xl md:block"><b>Address:</b> {{ address }}</n-p>
+        <n-p class="hidden text-xl md:block">
+          <b>Address:</b> {{ address }}
+          <n-button text class="my-2 mx-4" @click="navigateToEtherscan">
+            <n-icon class="mr-2" size="18px">
+              <svg
```
Comment (bran0h) [inline components/project/sections/NftInfo.vue]: Couldnt we use some icon from library?

**Review by bran0h — CHANGES_REQUESTED** [review]

File: components/project/sections/NftInfo.vue (hunk 3 above)
Comment (ktothova) [reply components/project/sections/NftInfo.vue]: didn't work sadly

File: components/project/sections/NftInfo.vue [hunk 4]
```diff
@@ -43,13 +48,17 @@
 
 <script setup lang="ts">
 import type { DataTableColumn } from 'naive-ui';
-import { NSkeleton, NDataTable } from 'naive-ui';
+import { NSkeleton, NDataTable, NButton } from 'naive-ui';
```
Comment (bran0h) [inline components/project/sections/NftInfo.vue:51]: Please remove naive component imports, they are auto-imported. Keep just type import

File: .vscode/settings.json [hunk 5]
```diff
@@ -1,8 +1,5 @@
 {
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.organizeImports": "never"
-  },
+  "editor.codeActionsOnSave": ,
```
Comment (bran0h) [inline .vscode/settings.json]: Please update from main branch

**Review by bran0h — CHANGES_REQUESTED** [review]

**Review by bran0h — APPROVED** [review]

**Pull request merged**